### ISSUE TITLE
Program allows user to Map Your Data when it has already been merged

### ISSUE THREAD

**Issue by RDmitchell** [SEED-platform/seed#1147]
In the past, if data has already been merged, the program greys out the option to Map Your Data, even though the user can view the mapping. 

Presumably this is partially due to the fact that the program doesn't close the Matching Progress modal, so it's not clear to the user when the program has actually finished matching the data, and I could click Cancel thinking I was cancelling the matching when really it was already completed. See #1138 

I am going to assign this to myself to test after #1138 has been fixed. Once that is fixed, hopefully it will not be possible to reproduce this.

Steps to reproduce this. 

* Import and map a file
* Click the Map Your Data button in the main mapping screen
* In the Mapping Review Screen, click the Save Mappings button
* Click Cancel in the Progress bar
* Go back to the Mapping screen
* Click the Map Your Data button 
* If you get this far, you should see the following screen
![image](https://cloud.githubusercontent.com/assets/6314950/20899057/a6c73ec8-badd-11e6-9aae-426b20a2ea6e.png)

**Comment by RDmitchell** [comment]
In V 1.5, which we should emulate, when you go back to data that has already been mapped, you can get as far as the Mapping Review screen, but on that screen there is no button to "Save Mapping".

V 1.5 -- This data has already been mapped.
You can click on Data Mapping for the data file to get to the mapping screen.
![image](https://cloud.githubusercontent.com/assets/6314950/20899739/83ed6406-bae0-11e6-857c-370e7fa9488d.png)

V 1.5 -- when you click the Map Your Data button, the program displays the Mapping Review screen. But there is no "Save Mapping" button.

**Comment by RDmitchell** [comment]
Decided today in iteration meeting it doesn't make sense to allow the user to go to the mapping review screen. Just going to see the field mappings (without showing the associated data) is good enough.

**Comment by RDmitchell** [comment]
Am closing this -- program doesn't show Mapping review for already mapped data (it goes to a screen with no data in it, however, which I will log as a separate issue).
